docs/specs/layers.md

```diff
@@ -12,6 +12,11 @@ The `ty` property defines the specific layer type based on the following values:
 
 {schema_subtype_table:layers/all-layers:ty}
 
+The `st` property specifies a start time offset, while `sr` defines a time stretch factor,
+to be applied when evaluating animated properties pertaining to the layer:
+
+$t\prime = \dfrac{t}{stretch} - start$
```

Comment:
May be worth explicitly calling out that values <1 increase the speed while >1 decrease the speed.

Reply:
Done.